src/components/Home/home.js

```diff
@@ -0,0 +1,62 @@
+import React, { Fragment, useState } from 'react';
+import { Link, Redirect } from 'react-router-dom';
+import './home.css';
+import Input from './Loginlist/input/input';
+import Label from './Loginlist/label/label';
+
+const Home = () => {
+  const [password, setPassword] = useState('');
```

Comment:
@Lizhtml @Yesika-Castillo I recommend you renaming this to something else as we're not talking about `passwords` here. It could be just `token` or `userList` (naming things is hard, I know)

Reply:
Listo ! :+1: